### DISCUSSION TITLE
setting multiple labels for an item

### DISCUSSION THREAD

**Discussion by Hildepont-0** [LeMyst/WikibaseIntegrator#105] [Q&A] [answered]
Is there a way to add multiple labels (from different languages) for an item ?
I tried this : 

> login_bot = wbi_login.Login(user='WikibaseAdmin@bot_1', pwd='XXX')
> data_z = []
> data_obj = wbi_core.String('test','P11')
> data_z.append(data_obj)				
> new_item = wbi_core.ItemEngine(data=data_z)
>
> new_item.set_label('en français','fr','REPLACE')
> new_item.set_label('in english','en','REPLACE')
> new_item.write(login_bot)

but I only get the last label ('in english')

**Comment by LeMyst** [comment]
I'm working on it in #107 
You can temporary use branch 'fix-if-exists-issue', I need to investigate if the issue is not somewhere else.

**Comment by LeMyst** [comment, marked as answer]
Fix merged in master

**Reply by Hildepont-0** [reply]
it works just fine, tx !
